.github/workflows/build-and-deploy.yml

```
@@ -0,0 +1,71 @@
+name: Build and Deploy
+
+on:
+    push:
+        branches: [master]
+    workflow_dispatch:
+
+jobs:
+    build-and-deploy:
+        name: 'Build userscripts and deploy to dist branch'
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout master branch
+              uses: actions/checkout@v5
+              with:
+                  fetch-depth: 0
+
+            - name: Setup Node.js
+              uses: actions/setup-node@v5.0.0
+              with:
+                  node-version: 20
+                  cache: 'npm'
+
+            - name: Install dependencies
+              run: npm install
```

BTW: it's probably better to just setup according to the lock file (ditto for `checks.yml` that has been using install for a long time) 
```suggestion
              run: npm ci
```

---

Okay, addressed.